src/main/java/LoginAction.java

```diff
@@ -0,0 +1,34 @@
+public class LoginAction {
+    private DataBase wishlistDatabase;
+//    private String username;
+//    private String password;
+
+    public LoginAction(DataBase wishlistDatabase){
+        this.wishlistDatabase = wishlistDatabase;
+//        this.username = username;
+//        this.password = password;
+    }
+
+    // check if the user exists in wishlist DataBase
```

Comment:
instead of comments above the functions, can u write docstrings instead for checkUsername and checkUserMatchesPassword? if u need an example of a docstring for a function look at the Wishlist class in my pull request. also delete the commented out code above.

Reply:
Thanks, I added the docstrings in my most recent commit!